documentation/asciidoc/accessories/audio/configuration.adoc

```diff
@@ -152,6 +154,8 @@ pcm.!default {
 
 Press `Ctrl+X`, then the `Y` key, then *Enter* to save. Reboot once more to complete the configuration:
 
+Note that the desktop environment in modern Linux distributions such as Raspberry Pi OS will use PulseAudio or PipeWire for audio control. These frameworks are capable of mixing and switching audio from multiple sources and they provide a high level API for audio applications to use. Out of the box, a lot of audio apps talk to Pulse or Pipe by default and the Pulse/Pipe lower layers communicate with ALSA. Creating ```~/.asoundrc``` is only recommended if audio applications are communicating directly with ALSA or running in an environment where Pulse or Pipe are not present. This file is not recommended if running the Raspberry Pi OS desktop environment because it can interfere with the UI's view of the underlying audio resources. The UI may take action to automatically clean up and remove this file if it exists so careful use of it is recommended, especially if it's intended to set any ALSA defaults.
```

Comment:
Seeing these shortened to just "Pulse" and "Pipe" looks a bit weird IMHO?

Reply:
:point_up: Still an issue.

Reply:
argh - not sure what happened there :-O
I think there are too many references to pulseaudio and pipewire so I'll reword it.

Reply:
Looks great now :+1:

Reply:
phew :-)